File: structures-js/structures-api/src/api/domain/Application.ts
import { Identifiable } from '@kinotic/continuum-client'
export class Application implements Identifiable<string> {
    public id: string
    public description: string
    public updated: number | null = null;
    public enableGraphQL: boolean = false
    public enableOpenAPI: boolean = false

    constructor(id: string, description: string) {
        this.id = id;
        this.description = description;
    }

}

File: structures-core/src/main/java/org/kinotic/structures/api/domain/Application.java
package org.kinotic.structures.api.domain;

import lombok.*;
import lombok.experimental.Accessors;
import org.kinotic.continuum.api.Identifiable;


import java.util.Date;

@Getter
@Setter
@Accessors(chain = true)
@NoArgsConstructor
public class Application implements Identifiable<String> {

    private String id;

    private String description;

    private Date updated = null;

    private boolean enableGraphQL = true;

    private boolean enableOpenAPI = true;

    public Application(String id, String description) {
        this.id = id;
        this.description = description;
    }
}
